src/catalog/postgres/builder.cpp

```diff
@@ -294,6 +302,33 @@ Schema Builder::GetTypeTableSchema() {
   return Schema(columns);
 }
 
+Schema Builder::GetLanguageTableSchema() {
+  std::vector<Schema::Column> columns;
+
+  columns.emplace_back("lanoid", type::TypeId::INTEGER, false, MakeNull(type::TypeId::INTEGER));
+  columns.back().SetOid(LANOID_COL_OID);
+
+  columns.emplace_back("lanname", type::TypeId::VARCHAR, MAX_NAME_LENGTH, false, MakeNull(type::TypeId::VARCHAR));
+  columns.back().SetOid(LANNAME_COL_OID);
+
+  columns.emplace_back("lanispl", type::TypeId::BOOLEAN, false, MakeNull(type::TypeId::BOOLEAN));
+  columns.back().SetOid(LANISPL_COL_OID);
+
+  columns.emplace_back("lanpltrusted", type::TypeId::BOOLEAN, false, MakeNull(type::TypeId::BOOLEAN));
+  columns.back().SetOid(LANPLTRUSTED_COL_OID);
+
+  columns.emplace_back("lanplcallfoid", type::TypeId::INTEGER, false, MakeNull(type::TypeId::INTEGER));
+  columns.back().SetOid(LANPLCALLFOID_COL_OID);
+
+  columns.emplace_back("laninline", type::TypeId::INTEGER, false, MakeNull(type::TypeId::INTEGER));
+  columns.back().SetOid(LANINLINE_COL_OID);
+
+  columns.emplace_back("lanvalidator", type::TypeId::INTEGER, false, MakeNull(type::TypeId::INTEGER));
+  columns.back().SetOid(LANVALIDATOR_COL_OID);
```

Comment:
These columns are not set in `InsertLanguage`.  Additionally, you have marked them as not nullable but are not setting them (@mbutrovich, is there a graceful way to catch this kind of behavior with asserts?).  Looking at Postgres' documentation, I think `lanispl = FALSE` and `lanpltrusted = TRUE` are reasonable initialization values (still leave the default as `NULL`) and the rest should probably be `NULL` and nullable until we go to implement them.